charts/dapr/charts/dapr_rbac/templates/sentry.yaml

```diff
@@ -61,6 +61,9 @@ rules:
   - apiGroups: ["dapr.io"]
     resources: ["configurations"]
     verbs: ["list"]
+  - apiGroups: ["authentication.k8s.io"]
+    resources: ["tokenreviews"]
+    verbs: ["create"]
```

Comment:
Token reviews are not namespaced and must be cluster role bound.

Reply:
Yes and I was waiting for this to cause me an issue with validation... not seen anything yet!

Reply:
Yes, today this won't do much but will cause issues in 1.12 when clients begin requesting their certs from sentry for real.